helpers.go

```diff
@@ -57,16 +57,22 @@ var formatFromExt = map[string]Format{
 	".gif":  GIF,
 }
 
-// FormatFromFilename parses image format from filename extension:
+// FormatFromExtension parses image format from extension:
 // "jpg" (or "jpeg"), "png", "gif", "tif" (or "tiff") and "bmp" are supported.
-func FormatFromFilename(filename string) (Format, error) {
-	ext := strings.ToLower(filepath.Ext(filename))
-	if f, ok := formatFromExt[ext]; ok {
+func FormatFromExtension(ext string) (Format, error) {
+	if f, ok := formatFromExt["."+strings.ToLower(strings.TrimPrefix(ext, "."))]; ok {
```

Comment:
I think it's better to remove a leading "." from the map keys (`formatFromExt`), so we don't need to add it here.